packages/kit/src/views/SignatureConfirm/components/TxFee/TxFeeInfo.tsx

```
@@ -970,11 +970,13 @@ function TxFeeInfo(props: IProps) {
 
     const decodedTx = decodedTxs[0];
 
-    let isInsufficientNativeBalance = false;
-    if (!decodedTx.isPsbt) {
-      isInsufficientNativeBalance = nativeTokenTransferAmountToUpdate.isMaxSend
+    let isInsufficientNativeBalance =
+      nativeTokenTransferAmountToUpdate.isMaxSend
         ? false
         : requiredNativeBalance.gt(nativeTokenInfo.balance ?? 0);
+
+    if (decodedTx && decodedTx.isPsbt) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)